index.bs

```diff
@@ -638,7 +650,9 @@ spec: secure-contexts; urlPrefix: https://www.w3.org/TR/secure-contexts/
 
       <p>
         The <dfn for='AudioConfiguration' dict-member>channels</dfn> member
-        represents the audio channels used by the audio track.
+        represents the audio channels used by the audio track. channels is only
```

Comment:
Channels is also applicable to to media-source and file usage.

Reply:
Done!